Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/hashtablesransomnote/HashTablesRansomNote.java

```diff
@@ -14,10 +14,6 @@ public static void main(String[] args) throws IOException {
         String blankSpace = " ";
         String[] firstMultipleInput = bufferedReader.readLine().replaceAll(regex, "").split(blankSpace);
 
-        int wordsInMagazine = Integer.parseInt(firstMultipleInput[0]);
-
-        int wordsInNote = Integer.parseInt(firstMultipleInput[1]);
-
         List<String> magazine = Arrays.asList(bufferedReader.readLine().replaceAll(regex, "").split(blankSpace));
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...